### ISSUE TITLE
vignette("treescape") does not work

### ISSUE THREAD

**Issue by thibautjombart** [thibautjombart/treescape#66]
The vignette does not seem to be found..
```r
> library(treescape)
Loading required package: ape
Loading required package: ade4
> vignette("treescape")
Warning message:
vignette ‘treescape’ not found 
> vignette(package="treescape")
no vignettes found
> 
```

**Comment by thibautjombart** [comment]
Odd because the vignette is indeed built and I can see it in the .tar.gz. ..

**Comment by thibautjombart** [comment]
@MichelleKendall I am still working on this; no idea why this is not working

**Comment by thibautjombart** [comment]
Fixed as of 895bd50
